src/autocomplete/popup.js

```diff
@@ -438,13 +438,18 @@ dom.importCssString(`
     background: #25282c;
     color: #c1c1c1;
 }
-.ace_autocomplete_right .ace_text-layer  {
-    width: calc(100% - 8px);
+.ace_autocomplete .ace_text-layer  {
+    width: 100%;
```

Comment:
Current: 
![with8](https://github.com/ajaxorg/ace/assets/12100596/33292792-a767-4108-888f-b7f222400c54)

After this PR:
![without8](https://github.com/ajaxorg/ace/assets/12100596/6810eb95-79e9-4ddc-a9f3-2b0c740f33b4)